libc/startup/baremetal/aie2/ldfile

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+ */
+
+MEMORY
+{
+   program (RX) : ORIGIN = 0, LENGTH = 0x0020000
+   data (!RX) : ORIGIN = 0x20000, LENGTH = 0x0020000
+}
+ENTRY(_main_init)
+SECTIONS
+{
+  . = 0x0;
+  .text : { 
+    *crt0.o(.text)
```

Comment:
don't we also need `*crt1.o(.text)` ?

Reply:
Also, `*` after the `.text`s ?

Reply:
We only need crt0.o to be in place, for loaders that expect the program to start at address 0. crt1 can be anywhere, it will be jumped to. Some might want to rely on fallthrough, not me.
Wildcarding text sections sounds like a good idea. My thought was that this is a vanilla for a vanilla invocation, and everything gets directed to .text if you don't pass extra options to the compiler.